app/assets/stylesheets/navbar_left.scss

```diff
@@ -179,6 +172,10 @@
     }
   }
 
+  #bitcoin_address {
```

Comment:
- Avoid using id selectors
- Selector `bitcoin_address` should be written in lowercase with hyphens